### ISSUE TITLE
WCAG compliance

### ISSUE THREAD

**Issue by felixgirault** [altcha-org/altcha#104]
Hello,

We're planning on using altcha on websites with strict accessibility requirements.
We ran an audit on the widget to ensure conformity (Using the RGAA, the official reference for french websites, which matches the WCAG). 
It scored really well but with those few exceptions:

### Contrast issue on the link to Altcha's website (minor)

The link "Protected by [ALTCHA](https://altcha.org/)" has a contrast of 2.8:1, while the minimum required is 4.5:1 

(WCAG https://www.w3.org/TR/WCAG22/#contrast-minimum)

### Contrast issue on the link to Altcha's website (minor)

[The logo](https://github.com/altcha-org/altcha/blob/main/src/Altcha.svelte#L1220) does not have an accessible name, thus making the surrounding link void.

(WCAG https://www.w3.org/TR/WCAG22/#non-text-content)

### Disappearing checkbox label (major)

At first, the checkbox is properly labeled.
When the user clicks it, the input disappears to make room for the status update (Verifying...).
Then the input comes back, but without labeling.

An option could be to always use the label to convey information (I'm not a robot/Verifying.../Verified).
Assistive technologies would announce text changes correctly, the input would always have a label, and the code would be simpler without the `aria-live` messages.
While verifying, the checkbox could be hidden visually while showing the loader animation.

(WGAG https://www.w3.org/TR/WCAG22/#labels-or-instructions)

---

I'd be glad to make a pull request if this can help, but I wanted to discuss the matter first :)

Have a good day

**Comment by ovx** [comment]
Hello, thank you for the suggestion. Using the label with the checkbox is a good solution. The logo and the link can be styled with custom CSS or hidden, but I agree, it should meet the criteria by default. You're welcome to create a PR, else I'll get to it later.

**Comment by ovx** [comment]
Accessibility improved in v2 with additional features. Users are recommended to update.
